person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require './nameable'
```

Comment:
- [ ] OPTIONAL
By convention, it's a good practice to use `require_relative` classes in the same source and use `require` for external modules or packages. Kindly ensure to improve upon this aspect.

Reply:
The codebase has been refactored to use `require_relative`. Thank you for the tip.